x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/use_host_isolation_status.tsx

```diff
@@ -33,16 +33,21 @@ export const useHostIsolationStatus = ({
   const { addError } = useAppToasts();
 
   useEffect(() => {
+    const abortCtrl = new AbortController();
     // isMounted tracks if a component is mounted before changing state
     let isMounted = true;
     const fetchData = async () => {
       try {
-        const metadataResponse = await getHostMetadata({ agentId });
+        const metadataResponse = await getHostMetadata({ agentId, signal: abortCtrl.signal });
         if (isMounted) {
           setIsIsolated(isEndpointHostIsolated(metadataResponse.metadata));
           setAgentStatus(metadataResponse.host_status);
         }
       } catch (error) {
+        // don't show self-aborted requests errors to the user
```

Comment:
I would argue that it is okay to show an info toast or some such message to the user when they do this. Hiding it would not give any feedback on the action performed. That's probably not something we want to do here?

Reply:
We discussed it briefly with @paul-tavares and decided to hide it. This error will show up everytime (and stack up in the notifications area) for each flyout open and canceled. We decided then to silence it.